sasl/digest/src/main/java/org/wildfly/security/sasl/digest/AbstractDigestMechanism.java

```diff
@@ -229,154 +229,153 @@ public byte[] wrap(byte[] outgoing, int offset, int len) throws SaslException {
-        }
+            byte[] cipheredPart = null;
+            try {
+                cipheredPart = wrapCipher.update(toCipher);
+            } catch (Exception e) {
+                throw saslDigest.mechProblemDuringCrypt(e).toSaslException();
+            }
+            if (cipheredPart == null){
+                throw saslDigest.mechProblemDuringCryptResultIsNull().toSaslException();
+            }
 
-        byte[] result = new byte[cipheredPart.length + 6];
-        System.arraycopy(cipheredPart, 0, result, 0, cipheredPart.length);
-        integerByteOrdered(1, result, cipheredPart.length, 2);  // 2-byte message type number in network byte order with value 1
-        integerByteOrdered(wrapSeqNum, result, cipheredPart.length + 2, 4); // 4-byte sequence number in network byte order
+            byte[] result = new byte[cipheredPart.length + 6];
+            System.arraycopy(cipheredPart, 0, result, 0, cipheredPart.length);
+            integerByteOrdered(1, result, cipheredPart.length, 2);  // 2-byte message type number in network byte order with value 1
+            integerByteOrdered(wrapSeqNum, result, cipheredPart.length + 2, 4); // 4-byte sequence number in network byte order
 
-        wrapSeqNum++;
-        return result;
-    }
+            wrapSeqNum++;
+            return result;
+        }
 
-    private byte[] unwrapConfidentialityProtectedMessage(byte[] message, int offset, int len) throws SaslException {
+        private byte[] unwrapConfidentialityProtectedMessage(byte[] message, int offset, int len) throws SaslException {
 
-        int messageType = decodeByteOrderedInteger(message, offset + len - 6, 2);
-        int extractedSeqNum = decodeByteOrderedInteger(message, offset + len - 4, 4);
+            int messageType = decodeByteOrderedInteger(message, offset + len - 6, 2);
+            int extractedSeqNum = decodeByteOrderedInteger(message, offset + len - 4, 4);
 
-        if (messageType != 1) {
-            throw saslDigest.mechMessageTypeMustEqual(1, messageType).toSaslException();
-        }
+            if (messageType != 1) {
+                throw saslDigest.mechMessageTypeMustEqual(1, messageType).toSaslException();
+            }
 
-        if (extractedSeqNum != unwrapSeqNum) {
-            throw saslDigest.mechBadSequenceNumberWhileUnwrapping(unwrapSeqNum, extractedSeqNum).toSaslException();
-        }
+            if (extractedSeqNum != unwrapSeqNum) {
+                throw saslDigest.mechBadSequenceNumberWhileUnwrapping(unwrapSeqNum, extractedSeqNum).toSaslException();
+            }
 
-        byte[] clearText = null;
-        try {
-            clearText = unwrapCipher.update(message, offset, len - 6);
-        } catch (Exception e) {
-            throw saslDigest.mechProblemDuringDecrypt(e).toSaslException();
-        }
-        if (clearText == null){
-            throw saslDigest.mechProblemDuringDecryptResultIsNull().toSaslException();
-        }
+            byte[] clearText = null;
```

Comment:
@keshav-redhat Just a minor, there seem to be some unneeded changes in this file now